.travis.yml

```diff
@@ -12,12 +12,17 @@ matrix:
     python: "nightly"
 
 before_install:
-  - pip install -U ".[dev]"
-  - pip install -U pytest-cov sphinx mypy
+  - pip install -U -r dev-requirements.txt
```

Comment:
Would it be better to add a `dev` extras which contains what is in the `dev-requirements.txt` so that we can drop this and have the `flit install --deps develop` cover it? My thinking is that this would keep everything in the `pyproject.toml` instead of keeping things split/duplicated across two files which is easy to forget to update one or the other.

Reply:
Yup. Adding `dev` extras and not having separate `dev-requirements.txt` sounds good to me.